src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -80,7 +81,7 @@ function MoneyRequestConfirmPage(props) {
 
     useEffect(() => {
         // ID in Onyx could change by initiating a new request in a separate browser tab or completing a request
-        if (prevMoneyRequestId.current !== props.iou.id) {
+        if (!isDistanceRequest && prevMoneyRequestId.current !== props.iou.id) {
```

Comment:
NAB: This !isDistanceRequest thing makes me think that distance requests could be broken easily with multiple tabs.

Reply:
yep exactly. I don't like my solution but it was the easiest at the moment 😓 

Let's definitely do a cleanup for this. The code is already confusing and I made it even confusing